matter_server/common/custom_clusters.py

```diff
@@ -99,6 +99,8 @@ def descriptor(cls) -> ClusterObjectDescriptor:
     wattAccumulatedControlPoint: float32 | None = None
     voltage: float32 | None = None
     current: float32 | None = None
+    altitude: int | None = None
```

Comment:
Altitude is in fact float32 too, so:

```suggestion
    altitude: float32 | None = None
```